src/librustc_unicode/u_str.rs

```diff
@@ -27,9 +27,16 @@ use core::str::Split;
 
 use tables::grapheme::GraphemeCat;
 
-/// An iterator over the words of a string, separated by a sequence of whitespace
+#[deprecated(reason = "struct Words is being replaced by struct SplitWhitespace",
+             since = "1.0.0")]
+#[unstable(feature = "unicode",
```

Comment:
This is wrong AFAICT. Stable deprecated items should just become deprecated, but not revert to unstable.